patches/components-crx_file-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/crx_file/BUILD.gn b/components/crx_file/BUILD.gn
+index 272b6f4f34e486804113475b9ae290b652236be8..b0fb4f603c196c86b1b6bffb275d03b70a789ded 100644
+--- a/components/crx_file/BUILD.gn
++++ b/components/crx_file/BUILD.gn
+@@ -75,7 +75,6 @@ proto_library("crx3_proto") {
+ 
+ if (host_toolchain == current_toolchain) {
+   executable("crx3_build_action") {
+-    testonly = true
```

Comment:
why this change was needed?

Reply:
Without this change, the build fails with:

```
ERROR at //build/config/BUILDCONFIG.gn:684:5: Test-only dependency not allowed.
    action("${target_name}") {
    ^-------------------------
//brave/build/mac:crx(//build/toolchain/mac:clang_x64)
which is NOT marked testonly can't depend on
//components/crx_file:crx3_build_action(//build/toolchain/linux:clang_x64)
which is marked testonly. Only targets with "testonly = true"
can depend on other test-only targets.

Either mark it test-only or don't do this dependency.
```

Reply:
Is this OK for you @goodov?

Reply:
> Is this OK for you @goodov?

Looks like chrome build this target separately and then runs outside of GN. For our approach we want to run it as a part of GN build flow, so we need to make it non-testonly. Which is fine.